pages/_main/_category/index.vue

```diff
@@ -3,9 +3,36 @@
   <div>
     <DHero
       variant="light"
-      :title="$route.params.category | humanize"
-      :subtitle="index.description"
+      :title="humanize($route.params.category)"
+      :subtitle="
+        list.find((x) => x.slug === $route.params.category).description
```

Comment:
this lookup is used in quite a few places - can it be pulled out to a computed variable? or at least a method?